### PR TITLE
fix(buySell): revert add recurring buy option in checkout page

### PR DESCRIPTION
**Pull request by dkremniov-bc**
## Description
This PR reverts commit 2045eab9 which has introduced a bug with recurring buys using BANK_TRANSFER payment method.

### CODE REVIEW

**Comment by milan-bc** [comment]
@dkremniov-bc great work so far, your title and. commit message are not in align with standards we do use

**Comment by blockdylanb** [comment]
This looks good to me, I'm going to hold off on approving/merging though. 4.73 was delayed until tomorrow so let's hold off of on merging this until it does to prod. Then we can merge this and follow up with a hot deploy so we can test it independently and not potentially delay the release any more.

**Review by blockdylanb — APPROVED** [review]

**Pull request merged**
